doc/whats_new/v1.5.rst

```diff
@@ -113,6 +113,13 @@ Changelog
 - |Enhancement| :term:`CV splitters <CV splitter>` that ignores the group parameter now
   raises a warning when groups are passed in to :term:`split`. :pr:`28210` by
 
+:mod:`sklearn.tree`
+...................
+- |Fix| :class:`tree.DecisionTreeRegressor` fixes an issue for ``squared_error``
```

Comment:
We need this fix for 1.4.1. You can merge just add your name to the already existing entry.

Reply:
Sure I moved it and edited the entry a bit to make it more informative